fish_update_completions: error when python not found

madblobfish opened the pull request:
## Description
I had a system without python and wondered why fish_update_completions was not working correctly, it exited quickly and printed nothing.
So I added a message for the user.

Added an option (-q/--quiet) which skips the message and returns 0.

## TODOs:
<!-- Just check off what what we know been done so far. We can help you with this stuff. -->
- [x] Changes to fish usage are reflected in user documentation/manpages.
- [x] Tests have been added for regressions fixed (not needed)
- [ ] User-visible changes noted in CHANGELOG.md (where exactly?)

floam commented:
The problem with this is we will automatically run `fish_update_completions` on first launch - it was decided that we didn't want to cause error output for an optional feature.

madblobfish commented:
I understand, how about passing a parameter to keep it quiet then?

floam commented:
That seems reasonable.

madblobfish commented:
Updated the implementation and added documentation.
Can't find where it is called during first launch, do you mean [this here](https://github.com/fish-shell/fish-shell/blob/master/share/functions/__fish_config_interactive.fish#L129)?

floam commented:
Yes, that's it.

floam commented:
Hm, so actually it won't run if python is not installed. I was mistaken.

floam commented:
Sooo... in retrospect maybe such a quiet option isn't really necessary at all. My bad!

floam commented on share/functions/fish_update_completions.fish:
```diff
@@ -7,5 +16,11 @@ function fish_update_completions --description "Update man-page based completion
+        if set -q _flag_quiet
+            return 0
+        end
+        printf "%s\n" (_ "python executeable not found")
```
Should be "executable"

madblobfish replied:
fixed

madblobfish commented:
Reverted to prior implementation without option.
I kept the help option though.

madblobfish commented:
Sorry, I didn't know that github responds so strange to rebase mistakes.
should be fine now

mqudsi commented:
Something killed the Travis CI build; I've restarted it manually though it'll take a bit of time to complete.
